### ISSUE TITLE
Scalar type appears before oneof type in generated case class, even when it's specified after the oneof

### ISSUE THREAD

**Issue by alexklibisz** [scalapb/ScalaPB#756]
First, fantastic project. I've used it for ~8 months in professional and personal projects and this is the first minor issue I've encountered.

I have a message like this:

```
message Foo {
    message Bar {
        int32 i = 1;
    }
    message Baz {
        int32 i = 1;
    }
    oneof inner {
        Bar bar = 1;
        Baz baz = 2;
    }
    bool flag = 3;
}
```

The generated case class looks like this:

```
@SerialVersionUID(0L)
final case class Foo(
    flag: _root_.scala.Boolean = false,
    inner: com.my.project.Foo.Inner = com.my.project.Foo.Inner.Empty
    ) extends scalapb.GeneratedMessage with scalapb.Message[Foo] with scalapb.lenses.Updatable[Foo] { ... }
```
I would have expected that the case class parameters follow the same order as the members of the message, so: `inner: com.my.project.Foo.Inner = ..., flag: _root_.scala.Boolean = ...`. It makes a difference in the ergonomics of the case class, since you always want to define `inner` but `flag` is `false` except in some very rare cases.

I'm using gradle with scalapb version 0.9.4.

**Comment by thesamet** [comment]
Hi @alexklibisz , this issue is resolved in 0.10.0 (you can check it with the latest 0.10.0-M5). This is listed as the 3rd bullet point here: https://github.com/scalapb/ScalaPB/blob/48ddc4109c376805ff0be7f85da029af43d5e1d1/CHANGELOG.md

Hope this helps!

**Comment by alexklibisz** [comment]
Fantastic. Thank you!
